### ISSUE TITLE
Ability to merge layers or choose multiple layers in layer blending

### ISSUE THREAD

**Issue by dfg138** [smeighan/xLights#2594]
**Is your feature request related to a problem? Please describe.**
Currently, layer blending is limited to effect 1 and all layers below combined (effect 2).  There are several scenarios I've run across where this could be helpful.  For example, say I have effects on layers 1-3 that I want to unmask layers 4-6.  There are many more complex examples where I'd want to groups of effects interacting with each other.  

**Describe the solution you'd like**
The ability to merge effects in multiple layers to create one effect or layer.  Similar to software like photoshop or gimp.  In the simple example above, Layers 1-3 could be merged to become a single effect 1, then unmask effect 2.  

**Describe alternatives you've considered**
Another option may be to allow an effect to blend with a chosen layer or group of layers below rather than only effect 2, which is a render of all subsequent layers.  

**Additional context**

**Comment by computergeek1507** [comment]
In the layer blending panel enable Canvas Mode, this then allows you to select which layers to blend.

![image](https://user-images.githubusercontent.com/2293065/114309747-4064b800-9ab6-11eb-90c0-a28f76782e18.png)

**Comment by dfg138** [comment]
Thank you Scott. I should have done a better job researching before submitting this enhancement request.  I will spend some time familiarizing myself with this tool.  I did take the example given above and I placed two off effects in layers above.  I used one off effect to group the 1st three effects, then another to group the 2nd three effects.  I then used the one of the off effects to unmask the other.  Is that the cleanest way of approaching this?
